### ISSUE TITLE
[BUG] webview_flutter

### ISSUE THREAD

**Issue by errajibadr** [sarbagyastha/youtube_player_flutter#778]
**Describe the bug**
Hello,

i have this issue on web. while using the package version youtube_player_iframe 4.0.2 .

My issue is quite simple. i tried to implement a simple youtube video but i got the following error :

**The following assertion was thrown building YoutubeVideoReader(dirty, state:
_YoutubeVideoReaderState#adc4d):
Assertion failed:
file:///Users/badrou/.pub-cache/hosted/pub.dev/webview_flutter_platform_interface-2.0.1/lib/src/platform_navigation_delegate.dart:35:7
../…/src/platform_navigation_delegate.dart:35
WebViewPlatform.instance != null
"A platform implementation for `webview_flutter` has not been set. Please ensure that an
implementation of `WebViewPlatform` has been set to `WebViewPlatform.instance` before use. For unit
testing, `WebViewPlatform.instance` can be set with your own test implementation."
The relevant error-causing widget was:
  YoutubeVideoReader**

**To Reproduce**
nothing. just followed the basics to test it.

```
  late final YoutubePlayerController _controller =
      YoutubePlayerController.fromVideoId(
    videoId: YoutubePlayerController.convertUrlToId(
        'https://www.youtube.com/watch?v=ZqbflgG55Lk')!,
    autoPlay: false,
    params: const YoutubePlayerParams(
      mute: false,
      showControls: true,
      showFullscreenButton: true,
    ),
  );

Widget build(BuildContext context) {
    return Container(
      child: YoutubePlayer(
        controller: _controller,
        aspectRatio: 16 / 9,
      ),
    );
}
```

**Expected behavior**
youtube video to play?

**Technical Details:**
 - Chrome

**Additional context**
i don't think there is any relevant context.
it's a very basic project where i try some packages.

thanks for your help!

**Comment by sarbagyastha** [comment]
Thanks for raising the issue. 
Published **v4.0.3** with the fix.

**Comment by errajibadr** [comment]
it's all good now.
